src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -705,6 +706,10 @@
       <field ref="remarks" in-xml="WITH_WRAPPER"/>
     </model>
     <constraint>
+      <allowed-values target="link/@rel" allow-other="yes">
+        &allowed-values-component_component_link-rel;
```

Comment:
I don't think these values belong here since they are intended to be used with the link on a component.

Is there a reason they were added here?

```suggestion
```